chromium_src/net/tools/transport_security_state_generator/input_file_parsers.cc

```diff
@@ -483,6 +483,18 @@ bool ParseJSON(base::StringPiece json,
       { "name": "static.ads.brave.com", "mode": "force-https", "policy": "custom", "pins": "brave"},
       { "name": "static.ads.bravesoftware.com", "mode": "force-https", "policy": "custom", "pins": "brave"},
 
+      // Search
+      { "name": "search.brave.com", "mode": "force-https", "policy": "custom", "pins": "brave"},
+      { "name": "cdn.search.brave.com", "mode": "force-https", "policy": "custom", "pins": "brave"},
+      { "name": "favicons.search.brave.com", "mode": "force-https", "policy": "custom", "pins": "brave"},
```

Comment:
![Screenshot from 2022-05-31 22-27-17](https://user-images.githubusercontent.com/167821/171334020-f99996b5-d67e-4965-beea-8bd251c5d656.png)

@remusao Is this hostname supposed to be reachable?

Reply:
Removed.